src/driver/oracle/OracleConnectionOptions.ts

```diff
@@ -16,6 +16,11 @@ export interface OracleConnectionOptions extends BaseConnectionOptions, OracleCo
      */
     readonly schema?: string;
 
+    /**
+    * A boolean determining whether to pass time values in UTC or local time. (default: true).
```

Comment:
isn't it a breaking change?

Reply:
there is no code in this PR where this option is set to `true` explicitly. Maybe comment is just wrong @Alankarsharma ?

Reply:
this is part of connection options. If user set this value to true in that case I am setting those environment variables.

Reply:
Default behaviour is false, so anyone who is using this for oracle or Postgres will not face any issue.

Reply:
so it will be great to change comments and documentation